pynamodb/attributes.py

```diff
@@ -406,6 +406,7 @@ def __init__(self, hash_key=False, range_key=False, null=None, default=None, att
                                            null=null,
                                            default=default,
                                            attr_name=attr_name)
+        self._is_raw_map = bool(type(self) == MapAttribute)
```

Comment:
Hm... I've been using it always that way without thinking about it.

The constant objects like `True` `False` `None` are compared via `is`. `type` is a reference to the same object, and you want to be sure it is the very same object, so `is` feels right to me there. But... I dont insist on it that much.

```python
class Foo(dict):
    pass

a = Foo()
b = Foo()
id(type(a))
# -> 28865504
id(type(b))
# -> 28865504
```